terminus-ui/login-form/src/login-form.component.html

```diff
@@ -58,7 +58,7 @@
       buttonType="submit"
       [showProgress]="inProgress || isRedirecting"
       [isDisabled]="!loginForm?.valid"
-      (clicked)="submit.emit(loginForm?.value)"
+      (clicked)="submission.emit(loginForm?.value)"
```

Comment:
Changing this will result in a breaking change rather than a deprecation. During the deprecation period, we need the emitter for both `submit` and `submission` to be fired for the same click.

Reply:
🤦‍♀